test/NullPointer.java

```diff
@@ -17,6 +17,12 @@ private static void throwAndCatch(Object o) {
 
   public static void main(String[] args) {
     try {
+      ((Object) null).getClass();
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+
+    try {
```

Comment:
Wait, why #396 wasn't caught here before? It looks exactly like a trap presented in the new test case.